examples/06-verif-manual/vm-007-plastic_compression_of_a_pipe_assembly.py

```diff
@@ -0,0 +1,581 @@
+# Define Section
+# ~~~~~~~~~~~~~~
+# Set up the cross-section properties for a shell and pipe elements.
+
+# Shell cross-section for inside (STEEL) tube.
+mapdl.sectype(1, "SHELL")
+
+# Thickness (SHELL181)
+mapdl.secdata(0.5, 1, 0, 5)
+
+# Shell cross-section for outside (Aluminum) tube.
+mapdl.sectype(2, "SHELL")
+
+# Thickness (SHELL181)
+mapdl.secdata(0.5, 2, 0, 5)
+
+# Define Pipe cross-section for inside (STEEL) tube, where:
+mapdl.sectype(3, "PIPE")
+
+# Outside diameter and wall thickness settings for Inside Tube (PIPE288)
+mapdl.secdata(4.9563384, 0.5)
+
+# Pipe cross-section for outside (Aluminum) tube.
+mapdl.sectype(4, "PIPE")
+
+# Outside diameter and wall thickness settings for Outside Tube (PIPE288)
+mapdl.secdata(8.139437, 0.5)
+
+# Print the section properties for all sections.
+print(mapdl.slist())
+
+
+###############################################################################
+# Define Geometry
+# ~~~~~~~~~~~~~~~
+# Set up the nodes and create the elements through the nodes.
+
+# Generate nodes and elements for PIPE288.
+mapdl.n(1, x=0, y=0, z=0)
+mapdl.n(2, x=0, y=0, z=10)
+
+# Create element for steel(inside) tube cross-section.
+mapdl.mat(1)
+mapdl.secnum(3)
+mapdl.e(1, 2)
+
+# Create element for aluminum(outside) tube cross-section.
+mapdl.mat(2)
+mapdl.secnum(4)
+mapdl.e(1, 2)
+
+# Activate the global cylindrical coordinate system.
+mapdl.csys(1)
+
+# Generate nodes and elements for SOLID185.
+mapdl.n(node=101, x=1.9781692)
+mapdl.n(node=101, x=1.9781692)
+mapdl.n(node=102, x=2.4781692)
+mapdl.n(node=103, x=3.5697185)
+mapdl.n(node=104, x=4.0697185)
+mapdl.n(node=105, x=1.9781692, z=10)
+mapdl.n(node=106, x=2.4781692, z=10)
+mapdl.n(node=107, x=3.5697185, z=10)
+mapdl.n(node=108, x=4.0697185, z=10)
+
+# Generate 2nd set of nodes to form a theta degree slice.
+mapdl.ngen(itime=2, inc=10, node1=101, node2=108, dy=theta)
+
+# Rotate nodal coordinate systems into the active system.
+mapdl.nrotat(node1=101, node2=118, ninc=1)
+
+# Create elements for Inside (Steel) tube.
+mapdl.type(2)
+mapdl.mat(1)
+mapdl.e(101, 102, 112, 111, 105, 106, 116, 115)
+
+# Create elements for Outside (Aluminum) tube
+mapdl.mat(2)
+mapdl.e(103, 104, 114, 113, 107, 108, 118, 117)
+
+# Generate nodes.
+mapdl.n(node=201, x=2.2281692)
+mapdl.n(node=203, x=2.2281692, z=10)
+mapdl.n(node=202, x=3.8197185)
+mapdl.n(node=204, x=3.8197185, z=10)
+
+# Generate nodes to form a theta degree slice
+mapdl.ngen(itime=2, inc=4, node1=201, node2=204, dy=theta)
+
+# Create element for Steel(Inside) tube cross-section.
+mapdl.type(3)
+mapdl.secnum(1)
+mapdl.e(203, 201, 205, 207)
+
+# Create element for Aluminum(Outside) tube cross-section.
+mapdl.secnum(2)
+mapdl.e(204, 202, 206, 208)
+
+# Plot element model to demonstrate the axisymmetric element model.
+cpos = [(19.67899462804619, 17.856836088414664, 22.644135378046194),
+        (2.03485925, 0.21270071036846988, 5.0),
+        (0.0, 0.0, 1.0)]
+_ = mapdl.eplot(cpos=cpos)
```

Comment:
I'm not sure this plot is relevant. Since we do not have (I believe) plotting capabilities for beams and pipes, I would delete any plot that relates to these elements.

![image](https://user-images.githubusercontent.com/28149841/149319157-c8ae0c30-efe9-4609-bfd2-371dd0e33981.png)

Reply:
> I'm not sure this plot is relevant. Since we do not have (I believe) plotting capabilities for beams and pipes, I would delete any plot that relates to these elements.
> 
> ![image](https://user-images.githubusercontent.com/28149841/149319157-c8ae0c30-efe9-4609-bfd2-371dd0e33981.png)


Thank you(@germa89) for your review.



May I ask you why you don't like to show axisymmetric model plot ?

I mean, from my point of view it is necessary to show that how we are simplifying the model and it is really needed to demonstrate it 

because in APDL you can see your model and check how it looks like.

Reply:
As we have been discussing it, I will leave it as it has got the same view of axisymmetric model as it has got APDL, you can see it below:
![image](https://user-images.githubusercontent.com/44849889/149354569-f023af6b-c8f7-46e8-a487-c1ddc0684950.png)